e2e/pages/StoryTask.js

```diff
@@ -26,6 +26,7 @@ class StoryTask extends Page {
     clickSaveStoryChanges() {
         CommonActions.waitAndClick(this.saveStoryChanges);
     }
+    //TODO change input for a json value
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=javascript%3AS1135)